srcpkgs/minikube/template

```diff
@@ -24,6 +25,13 @@ pre_configure() {
 
 pre_build() {
 	make pkg/minikube/assets/assets.go pkg/minikube/translate/translations.go
+
+	# Set up correct commit information to be embedded in the CLI.
+	_tmpdir=$(mktemp -d)
+	git clone --depth 1 --branch v${version} https://github.com/kubernetes/minikube ${_tmpdir}
+	cd ${_tmpdir}
+	go_ldflags+=" -X k8s.io/minikube/pkg/version.gitCommitID=$(git rev-parse --short HEAD)"
+	rm -rf ${_tmpdir}
```

Comment:
I would simply set `-X k8s.io/minikube/pkg/version.gitCommitID=v${version}`

Reply:
I'd rather have no commit information then. I mean, version and commit are meant to be different things in this case. Using `--depth 1` and `--branch` was the best way I found to accomplish fetching such information without impacting much in build time while not having to manually update the commit hash. Is it that bad? 😞

Reply:
These informations are not mandatory for compiling and running.

Reply:
Sure, I just wanted to improve information in the binary distributed by Void, although my solution was not ideal.